python/ucxx/ucxx/_lib_async/tests/test_from_worker_address.py

```diff
@@ -90,11 +91,9 @@ def test_from_worker_address():
     )
     client.start()
 
-    client.join()
-    server.join()
-
-    assert not server.exitcode
-    assert not client.exitcode
+    join_processes([client, server], timeout=30)
```

Comment:
**Problem**: The `join_processes` function call has a hardcoded timeout of 30 seconds, which may not be sufficient for all test cases.

**Suggested Change**: Consider making the timeout configurable or using a more robust method to wait for the processes to finish.


**Line**: 94

```suggestion
    join_processes([client, server], timeout=60)  # Increase timeout to 1 minute
```

-----

<div align="right">
    <small><i>Generated by <b>Llama 3.1 405b</b></i></small>
</div>
<br />


<div align="right">
  <small>Was this helpful?</small>
  <a href="https://codecritic-feedback.nvidia.com?vote=1&comment_guid=1faba244-f964-46e3-b45a-94b5759c452b&is_github=true" target="_blank">
    <button>👍</button>
  </a>
  <a href="https://codecritic-feedback.nvidia.com?vote=-1&comment_guid=1faba244-f964-46e3-b45a-94b5759c452b&is_github=true" target="_blank">
    <button>👎</button>
  </a>
</div>